cortexm_common.gpr

```diff
@@ -1,77 +1,21 @@
+with "config/cortex_m_config.gpr";
 abstract project CortexM_Common is
 
-   for Library_Version use "0.1.0";
+   for Library_Version use Project'Library_Name & ".so." & Cortex_M_Config.Crate_Version;
+
+   for Object_Dir use "obj/" & Cortex_M_Config.Build_Profile;
```

Comment:
This feature is not in the latest stable release yet ^^

Reply:
I was afraid you'd say that. I've removed the `Build_Profile` change for now, but kept the new Library_Version as alire 1.1 does appear to populate the `Crate_Version` value. The library version doesn't really matter, because nobody's running shared libraries on Cortex-M platforms, but maybe someday we can have an ELF loader.

Let me know when this diff looks good and I'll squash it down to one commit.